.github/workflows/build-mkdocs-commit-html.yml

```diff
@@ -10,7 +10,8 @@ on:
 
 jobs:
   docs:
-    runs-on: ubuntu-latest
+    runs-on:
+      labels: ubuntu-22.04-4core
```

Comment:
Is this the smallest machine available?

Reply:
Using larger runner yes, see: https://docs.github.com/en/actions/using-github-hosted-runners/using-larger-runners#machine-specs-for-larger-runners

Reply:
In that case, I'd suggest moving to a smaller one.  I don't think mkdocs needs a larer runner.